index.md

```diff
@@ -1,30 +1,32 @@
-### Introduction
+### OpenTK
 
-The Open Toolkit is set of fast, low-level C# bindings for OpenGL, OpenGL ES and OpenAL. It runs on all major platforms and powers hundreds of apps, games and scientific research.
+The Open Toolkit is set of fast, portable, low-level C# bindings for OpenGL, OpenGL ES, OpenAL, and OpenCL.  It runs on all major platforms and powers hundreds of apps, games, and scientific research programs.
 
 OpenTK provides several utility libraries, including a math/linear algebra package, a windowing system, and input handling.
 
-### Developer decisions
+OpenTK is not a game engine, nor does it try to be one:  It offers a useful set of tools and lets you decide how to use them.  It is the ideal starting point if you want to write a game or scientific application from the ground up.
 
-OpenTK is not a game engine, nor does it try to be one. It offers a useful set of libraries, and leaves it up to the developer to make the decisions on how to use them.
+### Current Status
 
-It is the ideal starting point if you want to write a game or scientific application from the ground up.
+- **OpenTK 4.x** is the current, stable version.  It targets .NET Core 3.x and .NET 5+.
+   - [OpenTK 4.x](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/v/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/dt/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/)
+   - [OpenTK 4.x GLWpfControl](https://www.nuget.org/packages/OpenTK.GLWpfControl/) (beta WPF control)
+   - [OpenTK 4.x GLControl](https://www.nuget.org/packages/OpenTK.WinForms) (beta WinForms control)
+- **OpenTK 3.x** is the legacy version.  It targets .NET Framework 4.x.
```

Comment:
It feels weird calling OpenTK 3 a legacy version as we are still merging PRs for it and releasing new versions. 
At least noting that people are welcome to submit OpenTK 3 PRs (and issues), and that we can release new versions is something we want.

Reply:
If OpenTK 5 is released later this year, what else would you call OpenTK 3 then?  Do we support all of OpenTK 3, 4, and 5 at that point?

Reply:
We would support OpenTK 3, and OpenTK 5. Basically, .net framework is going to live for a while so OpenTK 3 is also probably going to live for a while.

But sure, I think I would say "OpenTK 3 is for if you are stuck with .net framework" or something like that. But I guess using the term "legacy" is fine.